includes/Experiments.php

```diff
@@ -289,42 +289,6 @@ public function get_experiments(): array {
 				'group'       => 'editor',
 				'default'     => true,
 			],
-			/**
-			 * Author: @spacedmonkey
-			 * Issue: #11536
-			 * Creation date: 2022-06-22
-			 */
-			[
-				'name'        => 'posterHotlinking',
-				'label'       => __( 'Poster hotlinking', 'web-stories' ),
-				'description' => __( 'Enable story poster hotlinking', 'web-stories' ),
-				'group'       => 'editor',
-				'default'     => true,
-			],
-			/**
-			 * Author: @spacedmonkey
-			 * Issue: #11777
-			 * Creation date: 2022-06-24
-			 */
-			[
-				'name'        => 'linkIconHotlinking',
```

Comment:
I can still see `videoPosterHotlinking` and `linkIconHotlinking` defined in some Karma tests when doing a search in the whole codebase.